gitlab-pages.yaml

```diff
@@ -0,0 +1,42 @@
+package:
```

Comment:
```suggestion
# Source is on gitlab so we can't use github for updates
#nolint:git-checkout-must-use-github-updates
```

Reply:
this will silence the lint failure

Reply:
I added `package:` back which was missing.